OJP/OJP_Common.xsd

```diff
@@ -5,6 +5,13 @@
 	</xs:annotation>
 	<xs:import namespace="http://www.siri.org.uk/siri" schemaLocation="../siri/siri_model/siri_all-v2.0.xsd"/>
 	<xs:include schemaLocation="OJP_All.xsd"/>
+	<!-- ==== Entity ================================================================ -->
```

Comment:
Either remove this line or have it as annotation documentation like "General" in line 16